packages/web/src/components/notification/Notification/components/NotificationTile.tsx

```diff
@@ -31,7 +31,7 @@ export const NotificationTile = (props: NotificationTileProps) => {
   )
 
   return (
-    <Paper column shadow='flat' p='l' onClick={handleClick}>
+    <Paper column shadow='flat' p='l' onClick={handleClick} border='default'>
```

Comment:
looks unrelated. intended change?

Reply:
yeah unrelated, could make separate, but just going ahead with it